org.gnome.font-viewer.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "org.gnome.font-viewer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-font-viewer",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=ca.desrt.conf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
Do you need access to: 
"--filesystem=xdg-run/dconf",
"--filesystem=xdg-config/dconf:ro",

Reply:
This one, I am curious about - under Wayland, GTK+ directly reads GSettings (the ones that back  GtkSettings). So, while the font viewer does not itself use any settings, it still needs access. 

We clearly need a better story for dconf.

Reply:
Agree. Those four lines are the canonical examples from the docs though: http://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access but yeah, there's a reason why this upstream ticket has such a low number... https://github.com/flatpak/flatpak/issues/78